library/ssl_tls.c

```diff
@@ -5435,6 +5435,23 @@ int mbedtls_ssl_write_certificate( mbedtls_ssl_context *ssl )
     return( ret );
 }
 
+#if defined(MBEDTLS_SSL_RENEGOTIATION) && defined(MBEDTLS_SSL_CLI_C)
+static int ssl_check_peer_crt_unchanged( mbedtls_ssl_context *ssl,
+                                         unsigned char *crt_buf,
+                                         size_t crt_buf_len )
+{
+    mbedtls_x509_crt const * const peer_crt = ssl->session->peer_cert;
+
+    if( peer_crt == NULL )
+        return( -1 );
+
+    if( peer_crt->raw.len != crt_buf_len )
+        return( -1 );
+
+    return( memcmp( peer_crt->raw.p, crt_buf, crt_buf_len) );
```

Comment:
There should be a space between `crt_buf_len` and `)`.

Reply:
Thanks, I'll fix it.